webrender/src/renderer.rs

```diff
@@ -1076,6 +1077,7 @@ impl Renderer {
                    target_size: &DeviceSize,
                    cache_texture: Option<TextureId>,
                    should_clear: bool) {
+        let _ = GpuMarker::new("target draw");
```

Comment:
Oh, I missed the fact that those have names. Going to have the `add_marker` returning a `GpuMarker` object then, unless you think of a better way ;)